model/fs_page.php

```diff
@@ -190,7 +211,7 @@ public function all()
       $pagelist = $this->cache->get_array('m_fs_page_all');
       if( !$pagelist )
       {
-         $pages = $this->db->select("SELECT * FROM ".$this->table_name." ORDER BY lower(folder) ASC, lower(title) ASC;");
+         $pages = $this->db->select("SELECT * FROM ".$this->table_name." ORDER BY orden, lower(folder) ASC, lower(title) ASC;");
```

Comment:
Para solucionar lo del orden de los menús, mejor ordena primero usando folder, como estaba originalmente:
$pages = $this->db->select("SELECT * FROM ".$this->table_name." ORDER BY lower(folder) ASC, orden ASC, lower(title) ASC;");